news/_posts/core-weekly/2022-05-02-coreweekly-17-2022.md

```diff
@@ -0,0 +1,184 @@
+* [#28339](https://github.com/PrestaShop/PrestaShop/pull/28339): Bump version to 2.1.0 for `blockwishlist`, by [@Progi1984](https://github.com/Progi1984)
+* [#28274](https://github.com/PrestaShop/PrestaShop/pull/28274): Update guzzle psr7 to 2.2.1, by [@matks](https://github.com/matks)
+* [#28273](https://github.com/PrestaShop/PrestaShop/pull/28273): Update Symfony to 4.4.40, by [@matks](https://github.com/matks)
+* [#28197](https://github.com/PrestaShop/PrestaShop/pull/28197): Remove deprecated method inside SearchProvider. Thank you [@PrestaEdit](https://github.com/PrestaEdit)
+* [#28175](https://github.com/PrestaShop/PrestaShop/pull/28175): Add module name parameter to displayModuleConfigureExtraButtons hook, by [@sowbiba](https://github.com/sowbiba)
+
+
+### Back office
+* [#28319](https://github.com/PrestaShop/PrestaShop/pull/28319): BO: Fix overlapping checkoxes on configure buttons. Thank you [@hibatallahAouadni](https://github.com/hibatallahAouadni)
+* [#28314](https://github.com/PrestaShop/PrestaShop/pull/28314): Fix the stock supplier filter on the Stock page, by [@kpodemski](https://github.com/kpodemski)
+* [#28282](https://github.com/PrestaShop/PrestaShop/pull/28282): Add images selector to bulk combination form. Thank you [@zuk3975](https://github.com/zuk3975)
+* [#28235](https://github.com/PrestaShop/PrestaShop/pull/28235): Fix back button style on login page, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#28183](https://github.com/PrestaShop/PrestaShop/pull/28183): Add default group to customer listing. Thank you [@Hlavtox](https://github.com/Hlavtox)
+* [#28161](https://github.com/PrestaShop/PrestaShop/pull/28161): Add shop name in customer search result.. Thank you [@lmeyer1](https://github.com/lmeyer1)
+* [#28075](https://github.com/PrestaShop/PrestaShop/pull/28075): Migrate the Customers > Title pages (Only the list), by [@Progi1984](https://github.com/Progi1984)
+* [#28019](https://github.com/PrestaShop/PrestaShop/pull/28019): Fixed display of Vouchers id in Customer View, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Front office
+* [#27397](https://github.com/PrestaShop/PrestaShop/pull/27397): Add the minimum required to the error message, when trying to apply a cart rule with a minimum. Thank you [@Prestaplugins](https://github.com/Prestaplugins)
+
+
+### Installer
+* [#28258](https://github.com/PrestaShop/PrestaShop/pull/28258): Install theme before modules, by [@atomiix](https://github.com/atomiix)
+
+
+### Tests
+* [#28381](https://github.com/PrestaShop/PrestaShop/pull/28381): Fix error in title controller test, and fix a warning in module tests, by [@jolelievre](https://github.com/jolelievre)
+* [#27055](https://github.com/PrestaShop/PrestaShop/pull/27055): Split integration workflows into two, get rid of processIsolation, by [@jolelievre](https://github.com/jolelievre)
```

Comment:
Bonjour, sur la ligne 73, "processIsolation" est bien un terme utilisé ou il faudrait le séparer "process Isolation" ? Je rajoute ceci juste en commentaire. Merci !

Reply:
Thanks for the notice 👍  here we refer to a configuration item for PHPUnit
https://phpunit.readthedocs.io/en/9.5/configuration.html#the-processisolation-attribute

So the spelling is correct

Reply:
Perfect, thanks @matks for the answer